x-pack/plugins/maps/index.js

```diff
@@ -6,9 +6,9 @@
 
 import { resolve } from 'path';
 import { initRoutes } from './server/routes';
-import ecommerceSavedObjects from './server/sample_data/ecommerce_saved_objects.json';
-import fligthsSavedObjects from './server/sample_data/flights_saved_objects.json';
-import webLogsSavedObjects from './server/sample_data/web_logs_saved_objects.json';
+import { getSavedObjects as ecommerceSavedObjects } from './server/sample_data/ecommerce_saved_objects';
```

Comment:
why not just export the function name as `getEcommerceSavedObjects` to avoid export name conflicts? Then the exported names can just be used which will improve readability because `ecommerceSavedObjects ` does not sound like a function name